301-quarkus-vertx-kafka/pom.xml

```diff
@@ -17,14 +17,15 @@
         </repository>
     </repositories>
     <dependencies>
-        <dependency>
-            <groupId>io.quarkus</groupId>
-            <artifactId>quarkus-opentelemetry</artifactId>
-        </dependency>
-        <dependency>
-            <groupId>io.quarkus</groupId>
-            <artifactId>quarkus-opentelemetry-exporter-jaeger</artifactId>
-        </dependency>
+        <!-- TODO: https://github.com/quarkusio/quarkus/issues/17234 -->
```

Comment:
The same fix as in the module 300 done in https://github.com/quarkus-qe/beefy-scenarios/pull/206